packages/katello/katello/katello-debug.sh

```diff
@@ -52,6 +52,7 @@ add_files /var/log/tomcat*/host-manager*.log*
 add_files /var/log/tomcat*/localhost*.log*
 add_files /var/log/tomcat*/manager*.log*
 add_files /etc/candlepin/candlepin.conf
+add_files /etc/candlepin/broker.xml
```

Comment:
Do you also want the other files associated with auth?

https://github.com/theforeman/puppet-candlepin/blob/master/manifests/artemis.pp#L15-L39

@jturel

Reply:
Couldn't hurt

```
/etc/tomcat/cert-roles.properties
/etc/tomcat/cert-users.properties
/etc/tomcat/login.config
```

Reply:
@ehelms Done